crypto/fipsmodule/sha/sha3_test.cc

```diff
@@ -39,12 +39,26 @@ class SHA3TestVector {
     uint8_t digest[SHA3_256_DIGEST_LENGTH];
     EVP_MD_CTX* ctx = EVP_MD_CTX_new();
 
+    // Test the correctness via the Init, Update and Final Digest APIs.
     ASSERT_TRUE(EVP_DigestInit(ctx, algorithm));
     ASSERT_TRUE(EVP_DigestUpdate(ctx, msg_.data(), len_ / 8));
     ASSERT_TRUE(EVP_DigestFinal(ctx, digest, &digest_length));
     
-    // Alternatively, the correctness can be tested via the single shot API.
-    //ASSERT_TRUE(EVP_Digest(msg_.data(), len_ / 8, digest, &digest_length, algorithm, NULL));
+    ASSERT_EQ(Bytes(digest, SHA3_256_DIGEST_LENGTH),
+              Bytes(digest_.data(), SHA3_256_DIGEST_LENGTH));
+    
+    OPENSSL_free(ctx);
+  }
+
+    void NISTTestVectors_SingleShot() const {
+   
```

Comment:
This empty line is not needed.

Reply:
Thank you. I removed it, also removed it form the previous test function.